lib/stitches/generator_files/db/migrate/add_enabled_to_api_clients.rb

```diff
@@ -0,0 +1,7 @@
+class AddEnabledToApiClients < ActiveRecord::Migration
+  def change
```

Comment:
Cool - didn't know that was possible.  Improved the deprecation message with the "fix" as well.